code/modules/research/nanites/nanite_programs/suppression.dm

```diff
@@ -131,38 +134,19 @@
 	trigger_cost = 3
 	trigger_cooldown = 20
 	rogue_types = list(/datum/nanite_program/brain_misfire, /datum/nanite_program/brain_decay)
+	var/static/list/blacklist = ("*surrender")
```

Comment:
Make sure you document that you've enabled emotes in forced speech

Reply:
fair enough, I just wanted to put in *clap

Reply:
reminder to remove *collapse